puppet/modules/quickstack/manifests/cinder_volume.pp

```diff
@@ -32,6 +35,17 @@
   $eqlx_chap_login        = ['chapadmin'],
   $eqlx_chap_password     = [''],
 
+  $dell_sc_san_ip           = [''],
+  $dell_sc_san_logi         = ['Admin'],
+  $dell_sc_san_password     = [''],
+  $dell_sc_iscsi_ip_address = ['192.168.0.20'],
+  $dell_sc_iscsi_port       = ['3260'],
+  $dell_sc_ssn              = ['64702'],
+  $dell_sc_api_port         = ['3033'],
+  $dell_sc_server_folder    = ['server'],
+  $dell_sc_volume_folder    = ['volume'],
+
```

Comment:
san_thin_provision is parameter supported by EQLX. I believe thin provisioning will be part of consistency groups for Dell Storage Center